src/components/wallet/methods/signOut.ts

```diff
@@ -1,22 +1,13 @@
 import { remove } from '@services/storage'
 import { handleError } from '@services/error'
 import { Wallet } from '../wallet'
-import { PromptInput } from '../../prompt/promptInput'
 
 export default async function signOut(this: Wallet) {
   try {
-    const inputs = await this.setPrompt({
```

Comment:
IMO I don't think we should remove the confirmation prompt entirely.
It could be better to have a prompt that gives the user a warning and an opportunity to save their secretkey to a clipboard before their account gets _"nuked"_.

That being said remove the `"Enter NUKE to confirm"` part
but include a press enter to confirm
and add `copy secret key` to the same prompt.

Reply:
I replaced the text message with "You can press the back button to sign back in.", but right now we can't pass buttons to the prompt for copying the secret key. The secret key is also in the URL now so the button isn't as needed. LMK if this works.